CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,259 @@
+
+For the duration of this course, take "participation" to mean some combination of:
+
+* helping in group activities
+* completing assignments
+* Listens to class instruction
+* Giving full effort to class activities
+* Actively contributing to collaborative assignments
+* Being prepared for class by completing work outside of class hours
+* For group assignments, each member of the group should appear in the "Contributors" list
+* Respect neighborhood members' working hours
+  * If a member prefers not to work past 9:00p, recognize that their participation and time may be limited to a certain window each day and that this window may not always be the same
+  * However, make your best effort to be available to peers in your neighborhood rather than setting extremely restrictive hours
+
+Please note that none of the above definitions _requires verbal contribution_ -- while this is a _way_ to participate, it is not the _only_ way. Activity on GitHub may also constitute full participation, if efforts display the above qualities. Occasionally, for group assignments, your neighborhood may receive a level of effort survey to gauge peer appraisal of team contribution as well.
+
+As several of you note in your suggestions for this contract, it is good practice to consider your participation not only necessary for _your own success_, but _required for the success of your neighborhood as a whole_.
+
+#### Communicating in your neighborhoods
+
+The Office of the Mayor expects neighborhoods to use their GitHub neighborhood pages to coordinate working hours on desired platforms 
+
+#### Course platforms
+
+On course platforms used for the duration of the semester:
+
+* Use all platforms for appropiate class reasons 
+* Follow the instructions while using the platforms
+* Keeping "up-to-date" with platforms may require seeking additional extra-curricular knowledge
+  * For example, using Markdown in Discord
+* Respect others' thoughts, time, and contributions to either discussion or code product
+* Check course platforms often -- at least twice a day
+  * The best times to review are often the beginning and ends of a day
+    * i.e.: 8:00 (beginning), 18:00 (end)
+
+In addition, I suggest that you at least download and install the Discord application, rather than use the web interface. This is not required, but will help greatly with your ability to keep up with notifications.
+
+#### Using office hours
+
+When using office hours:
+
+* Come in prepared with questions, a topic, or a clear objective
+  * To achieve this, please add notes about expected topic(s) to a Google Calendar slot reservation (for instructor calendar)
+* Be prepared to receive feedback, understanding that typically this means completing revisions on what ever you are working on
+* Respect the time of other citizens (i.e. be mannerly and quick)
+* Have all assignments ready to discuss with the Professor
+* Not rely only on office hours (i.e. work should be attempted first)
```

Comment:
Perhaps have a way to contact our neighborhood members alongside office hours as most projects are collaborative